### DISCUSSION TITLE
docs/envoy-handbook/intro/envoy-building-blocks/

### DISCUSSION THREAD

**Discussion by giscus** [rootsongjc/cloud-native-library#11] [Announcements]
# docs/envoy-handbook/intro/envoy-building-blocks/

在这一节中，我们将解释 Envoy 的基本构建模块。 Envoy 配置的根被称为引导配置。它包含了一些字段，我们可以在这里提供静态或动态的资源和高级别的 Envoy 配置（例如，Envoy 实例名称、运行时配置、启用管理界面等等）。 为了开

https://jimmysong.io/docs/envoy-handbook/intro/envoy-building-blocks/

**Comment by rootsongjc** [comment]
本文适用于 Envoy 1.22 以下版本，请见[此 issue](https://github.com/rootsongjc/docs/issues/9)
